packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -52,11 +52,11 @@ class IOSEmulator extends Emulator {
 
     // First run with `-n` to force a device to boot if there isn't already one
     if (!await launchSimulator(<String>['-n']))
-      return false;
-    
+      return;
+
     // Run again to force it to Foreground (using -n doesn't force existing
     // devices to the foreground)
-    return launchSimulator(<String>[]);
+    await launchSimulator(<String>[]);
   }
```

Comment:
Shouldn't this `return true` now?

Reply:
See the comments above - I think these should all be `void`s, no bools anywhere. In the failure case we `throw` now so we don't use any return values.

Reply:
Ah, gotcha - yeah, all `Future<void>` sgtm

Reply:
See comment below - I think this is actually correct. `launchSimulator` returns a bool so we can tell if the first invocation succeeds, but we don't care about the result of the second one, we just wait for it to finish.